### DISCUSSION TITLE
Playback of tracks encoded with Dolby Atmos are too quiet

### DISCUSSION THREAD

**Discussion by expattaxtools** [music-assistant/support#1724] [Q&A] [answered]
Is it feasible to normalize the volume levels between Dolby Atmos (eac3) and stereo (flac) encoded tracks? Currently, the Atmos tracks are noticeably quieter than the stereo counterparts. Additionally, is it possible to set a default preference for playback when a track is available in both Dolby Atmos and stereo formats?

**Comment by OzGav** [comment]
@marcelveldt ??

**Reply by marcelveldt** [reply]
This is where the volume normalization feature is invented for. Only problem is that we do not pre-fetch that info. Its only measured after first playback. I guess we can add an option to proactively scan the items that are on disk

**Reply by OzGav** [reply, marked as answer]
@expattaxtools An issue has been raised to address this. Currently low priority but thanks for identifying it https://github.com/music-assistant/hass-music-assistant/issues/1961
